src/lib.rs

```diff
@@ -35,10 +35,41 @@ impl<'a> Image<'a> {
     }
 }
 
+/// Scales the alpha to the downscaled texture to preserve the overall alpha coverage.
+/// If alpha cutoff is specified, any alpha value above it is considered visible
+/// of which the percentage of visible texels will be.
+/// Otherwise, visibility is considered a linear sum of the alpha values instead
+/// and the source and target alpha coverage are calculated the same way
```

Comment:
Note that rustdoc will fold this onto a single line. Add an empty line (`///`) if you intend to separate this into paragraphs. And don't forget to finish the last sentence with a period.